architecture/core_concepts/routes.adoc

```diff
@@ -125,9 +125,10 @@ image::router_model.png[HAProxy Router Data Flow]
 Implementing sticky sessions is up to the underlying router configuration. The
 default HAProxy template implements sticky sessions using the `balance source`
 directive which balances based on the source IP. In addition, the template
-router plug-in provides the service name and namespace to the underlying
-implementation. This can be used for more advanced configuration such as
-implementing stick-tables that synchronize between a set of peers.
+router plug-in provides the service name (limited to 63 characters) and
```

Comment:
I think this mention can be dropped since you already mention the service name limit in the services topic.